packages/data-flow/src/types/index.ts

```diff
@@ -1,9 +1,27 @@
-import { Changeset } from "@grants-stack-indexer/repository";
+import { ProcessorDependencies } from "@grants-stack-indexer/processors";
+import {
+    Changeset,
+    IApplicationRepository,
+    IProjectRepository,
+    IRoundRepository,
+} from "@grants-stack-indexer/repository";
 
+/**
+ * The result of the execution of the changesets.
+ */
 export type ExecutionResult = {
     changesets: Changeset["type"][];
     numExecuted: number;
     numSuccessful: number;
     numFailed: number;
     errors: string[];
 };
+
+export type CoreDependencies = Pick<
+    ProcessorDependencies,
+    "evmProvider" | "pricingProvider" | "metadataProvider"
+> & {
+    roundRepository: IRoundRepository;
+    projectRepository: IProjectRepository;
+    applicationRepository: IApplicationRepository;
+};
```

Comment:
it's subtle but the difference is that Processor use the `Read` interface, ie. they don't know about write operations on the Repository, but on the Orchestrator we need Read and Write

(maybe was over-optimization at this point but when we implement read-only PG connections this will be useful)